### PR TITLE
Add Open Computational Neuroscience Resources

### PR DESCRIPTION
**Pull request by asoplata**
I would like to add my curated list of computational neuroscience resource tools. There is much fragmentation of methods and tools in the computational neuroscience community and this is an attempt to compile them. That said, this also means there are a great many ways anyone can contribute to the field or learn from it with nothing more than a computer! These are very accessible resources for anyone desiring to learn about the brain.

**https://github.com/asoplata/open-computational-neuroscience-resources**

**Neuroscience is a popular research topic these days, and computational neuroscience, both of simulating the activity of neural structures and the analysis of open neural data like EEG, fMRI, etc. recordings, is only going to become more important. Furthermore, computational neuroscience only requires a computer for someone to begin to learn about the field and contribute to it, unlike expensive wet lab research. There is a large number of computational neuroscience tools and data repositories that are publicly available, and I think many people would benefit from just knowing where to look for open and free guidance on this.**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

File: readme.md [hunk 1]
```diff
@@ -588,6 +588,7 @@
 - [Visual Regression Testing](https://github.com/mojoaxel/awesome-regression-testing) - Ensures changes did not break the functionality or style.
 - [Theravada](https://github.com/johnjago/awesome-theravada) - Teachings from the Theravada Buddhist tradition.
 - [inspectIT](https://github.com/inspectit-labs/awesome-inspectit) - Open source Java app performance management tool.
+- [Open Computational Neuroscience Resources](https://github.com/asoplata/open-computational-neuroscience-resources) - Computational neuroscience tools and data are much more accessible than you would think, and all anyone needs to learn from or contribute to the field is passion and a computer.
```
Comment (legodude17) [inline readme.md]: Explain what computational neuroscience is in this description

**Comment by asoplata** [comment]
The description of computational neuroscience should now be clear enough.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/asoplata/open-computational-neuroscience-resources
✖ Linting

  README.md:31:5
  ✖    1:1   Missing file contributing.md                                          remark-lint:awesome/contributing
  ✖    1:1   Missing License section                                               remark-lint:awesome/license
  ✖    1:1   Missing or invalid Table of Contents                                  remark-lint:awesome/toc
  ✖   28:1   Rules should use ---                                                  remark-lint:rule-style
  ✖   31:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖   31:7   Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   32:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖   32:7   Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   34:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖   34:7   Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   35:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖   35:7   Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   36:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖   36:7   Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   37:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖   37:7   Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   38:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖   38:7   Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖   40:1   Rules should use ---                                                  remark-lint:rule-style
  ✖   51:17  Invalid list item link                                                remark-lint:awesome/list-item
  ✖   57:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖   73:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  143:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  160:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  203:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  217:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  219:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  220:3   Link to http://bodb.usc.edu/bodb/ is dead                             remark-lint:no-dead-urls
  ✖  231:3   Invalid list item link URL                                            remark-lint:awesome/list-item
  ✖  252:3   Link to https://library.tmc.edu/website/whole-brain-catalog/ is dead  remark-lint:no-dead-urls
  ✖  261:54  List item link and description must be separated with a dash          remark-lint:awesome/list-item
  ✖  263:45  List item link and description must be separated with a dash          remark-lint:awesome/list-item
  ✖  265:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  269:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  273:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  277:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  280:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  284:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  286:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent
  ✖  287:5   Incorrect indentation before bullet: remove 2 spaces                  remark-lint:list-item-bullet-indent

  40 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
The item description should not be part of the link.

**Comment by sindresorhus** [comment]
I would rename the list `Open Computational Neuroscience Resources` => `Awesome Open Computational Neuroscience`.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**